package.json

```diff
@@ -199,6 +198,7 @@
     "metaviewport-parser": "0.2.0",
     "open": "^8.4.0",
     "parse-cache-control": "1.0.1",
+    "puppeteer": "^10.2.0",
```

Comment:
Since we do not need puppeteer's bundled chromium (we use chrome-launcher), we should use `puppeteer-core` instead.

https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#puppeteer-vs-puppeteer-core

Reply:
In fact, it might be worth first doing a PR that changes our devDep from pup to pup-core.

There's also #13657 to consider.

Reply:
In that case, we can keep `puppeteer` as a dev dep because we use it in a few scripts, and then just have pup-core as the package dep.